src/devices/sound/sn76477.cpp

```diff
@@ -894,7 +894,9 @@ void sn76477_device::open_wav_file()
 {
 	char wav_file_name[30];
 
-	sprintf(wav_file_name, LOG_WAV_FILE_NAME, tag());
+	std::string s = tag();
+	std::replace(s.begin(), s.end(), ':', '_');	
+	sprintf(wav_file_name, LOG_WAV_FILE_NAME, s.c_str());
```

Comment:
Yes, it could be done that way, but I preferred to keep the most of the pre-existent code and to change only the parameter of the sprintf. 
Anyway if you prefer I can change also the other parts of the code.

Reply:
Ok, I've used util::string_format as you suggested. Please look at my latest commit, thank you.